mapic/metrics/census.go

```diff
@@ -225,6 +237,41 @@ func InitCensus(nodeID, version, namespace string) {
 			TagKeys:     append([]tag.Key{Census.kType}, baseTags...),
 			Aggregation: view.Distribution(0, 0.050, 0.100, .250, .500, .750, 1.000, 1.250, 1.500, 2.000, 2.500, 3.000, 3.500, 4.000, 4.500, 5.000, 10.000, 20.0, 30.0, 60.0),
 		},
+		{
+			Name:        "log_processor_write_errors_count",
+			Measure:     Census.mLogProcessorWriteErrors,
+			Description: "Number of log processors errors while writing to Kafka",
+			TagKeys:     baseTags,
+			Aggregation: view.Count(),
+		},
+		{
+			Name:        "analytics_log_errors",
+			Measure:     Census.mAnalyticsLogErrors,
+			Description: "Number of errors while posting analytics log",
```

Comment:
Is it posting or fetching issue?
https://github.com/livepeer/catalyst-api/pull/1265/files#diff-9d26d9175b31b67c2fcc702fa795bafd990d74b3fcac9f55df02ca461b25f41fR119

Reply:
This is mostly the error with the request itself. Like the user posts the wrong `playback_id` or some other data is wrong. Not sure what will be the better name... any thoughts?